### PR TITLE
Develop.new chat sample fixes

### PR DESCRIPTION
**Pull request by IegorKozakov**
* Fixes with sending messages in a group chat.
* Add room name.
* Update dialog fixes.
* Users count on the create dialog page.

### CODE REVIEW

File: samples/newChat/css/dashboard.css [hunk 1]
```diff
@@ -52,6 +57,7 @@
     font-weight: 500;
     line-height: 20px;
     color: #4A4A4A;
+    margin: 0 0 6px;
```
Comment (dimaspirit) [inline samples/newChat/css/dashboard.css]: use 2 or 4 values

File: samples/newChat/js/route.js [hunk 2]
```diff
@@ -115,6 +114,9 @@ router.on({
                     });
 
                 }).catch(function(error){
+                    console.error(error);
```
Comment (dimaspirit) [inline samples/newChat/js/route.js:117]: Is this need?

Comment (IegorKozakov) [reply samples/newChat/js/route.js:117]: Yes. I use console.error() to notify developers about the errors in catch part of the promise.

File: samples/newChat/css/dashboard.css (hunk 1 above)
Comment (IegorKozakov) [reply samples/newChat/css/dashboard.css]: ok

**Pull request merged**